lib/ansible/modules/windows/win_format.ps1

```diff
@@ -81,7 +81,7 @@ function Get-AnsibleVolume {
     }
     elseif ($null -ne $Path) {
         try {
-            $volume = Get-Volume -Path $Path
+            $volume = Get-Volume -LiteralPath $Path
```

Comment:
I cannot find `-LiteralPath` in https://docs.microsoft.com/en-us/powershell/module/storage/get-volume?view=win10-ps.